django/river/settings/base.py

```diff
@@ -97,7 +97,7 @@
         "NAME": os.environ.get("POSTGRES_DB", "river"),
         "USER": os.environ.get("POSTGRES_USER", "river"),
         "PASSWORD": os.environ.get("POSTGRES_PASSWORD"),
-        "HOST": os.environ.get("POSTGRES_HOST"),
+        "HOST": os.environ.get("POSTGRES_HOST", "localhost"),
```

Comment:
Do you need to include that in there ?
It's unrelated to the feature, and you just have to use the `.env` file.